python/ray/dataframe/test/test_dataframe.py

```diff
@@ -1224,10 +1248,33 @@ def test_quantile():
 
 
 def test_query():
-    ray_df = create_test_dataframe()
+    def generate_data():
```

Comment:
I prefer these be in `@pytest.fixture` calls and pass the different dataframes to it.

Reply:
You can also pass in what you want to query for.

Reply:
Done.